module-apps/apps-common/popups/WindowWithTimer.hpp

```diff
@@ -8,26 +8,32 @@
 
 namespace gui
 {
-    inline constexpr auto defautTimeout   = std::chrono::seconds{3};
-    inline constexpr auto noTimeoutChange = std::chrono::seconds::zero();
     class WindowWithTimer : public gui::AppWindow
     {
-      private:
-        sys::TimerHandle popupTimer;
-        std::chrono::milliseconds timeout;
-
       public:
-        explicit WindowWithTimer(app::ApplicationCommon *app,
-                                 const std::string &name,
-                                 const std::chrono::milliseconds timeout = defautTimeout);
-        void destroyInterface() override;
+        static constexpr auto defaultTimeout  = std::chrono::seconds{3};
+        static constexpr auto noTimeoutChange = std::chrono::seconds::zero();
```

Comment:
is there a reason why these values are in the header file?

Reply:
They're used as default arguments to ctor and some method in this class :wink: I moved them inside this class as they have quite generic names to avoid some potential naming clash in the future